content/en/docs/reference/using-api/api-concepts.md

```diff
@@ -627,13 +631,84 @@ read-modify-write and/or patch are the following:
 It is strongly recommended for controllers to always "force" conflicts, since they
 might not be able to resolve or act on these conflicts.
 
+### Transferring Ownership
+
+In addition to the concurrency controls provided by [conflict
+resolution](#conflicts), Server Side Apply provides ways to perform coordinated
+field ownership transfers from users to controllers.
+
+First, the user delegates ownership of the field to a temporary, uniquely named,
+field manager. Next, the user signals to the controller that it should take
+ownership of the field, usually by setting some other field (perhaps in some
+other object) that the controller observes. Last, controller then claims ownership
+of the field by updating the object or applying a configuration containing the
+field.
+
+Transferring fields to controllers this way ensures the field is not accidently
+removed from the live object, or reset to a default value, at any point of time
+during the transfer.
+
+This approach can be used, for example, to safely transfer ownership of the
+`replicas` field from a user to a controller while enabling Horizontal Pod
+Autoscaling on an object.
+
+Say a user has defined deployment with `replicas` set to 100:
+
+{{< codenew file="application/ssa/nginx-deployment.yaml" >}}
+
+And the user created the deployment using server side apply like so:
+
+```shell
+kubectl apply -f application/ssa/nginx-deployment.yaml --server-side
+```
+
+Before the user enables Horizontal Pod Autoscaling, the user needs to transfer
```

Comment:
It's not before enabling HPA. You need to do this when all of 1) you enabled HPA, 2) HPA controller has not yet taken over the field, and 3) you need to make another change.